### ISSUE TITLE
assert gnorm >= 0, 'encounter nan in gradients

### ISSUE THREAD

**Issue by mustfkeskin** [hitvoice/RE2#3]
Hello

I get the following error when I try to train the model with my own training data.
What could be the error

![image](https://user-images.githubusercontent.com/5824559/69963392-31184680-1521-11ea-8952-b101f91852a2.png)

**Comment by hitvoice** [comment]
Can you get the batch which causes NaN? Maybe by catching the exception and save the batch data (feed_dict).

**Comment by mustfkeskin** [comment]
I got 1 empty string "" in my training data.
I drop it then it works

Thanks

![image](https://user-images.githubusercontent.com/5824559/70030234-d3840880-15b9-11ea-9b36-8c41b4eea95a.png)
